### DISCUSSION TITLE
Welcome to everforest-nvim Discussions!

### DISCUSSION THREAD

**Discussion by neanias** [neanias/everforest-nvim#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mstuttgart** [comment]
Hi! I'm Michell. Backend developer from Brazil. Everforest is a great theme, and I'm currently using it on nvim, vscode, alacritty, i3, polybar, Obsidian and Firefox etc. :)

I want to suggest create a GitHub organization to centralize all Everforest applications, like is made on [Catppuccin](https://github.com/catppuccin), [Nord](https://github.com/nordtheme) and [Rose Pine](https://github.com/rose-pine). 

This theme has a big potential and deserve more attention. Great port from original Everforest. Thanks to share it :)

**Reply by neanias** [reply]
That would be fun, but I feel a bit unsure about doing that given this is a port of the original theme. I think it would be better to have it driven from the upstream repo rather than downstream here.
